src/SensioLabs/Melody/Exception/AuthenticationRequiredException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SensioLabs\Melody\Exception;
+
+use SensioLabs\Melody\Handler\ResourceHandlerInterface;
+
+/**
+ * Class AuthenticationRequiredException.
```

Comment:
I hate so much this kind of PHPDoc. In melody, we never use PHPDoc, except if it's useful. 
So, please, remove all useless PHPDoc.
(useless === that can not be guess by a monkey)